src/core/agent/agent.types.ts

```diff
@@ -51,6 +51,15 @@ type ConnectionNoteDetails = {
   message: string;
 };
 
+interface IpexHistoryItem {
```

Comment:
this should be in `connectionService.types.ts` - also not sure why there's a `connection.types.ts` - can you move them all to the service type file?